### PR TITLE
Add endpoint to return the /excludes/ list from synclists.

### PR DESCRIPTION
**Pull request by rochacbruno**
pulp_ansible sync is going to try a fetch on `{REMOTE}/<path>/excludes/`
this endpoint returns the consolidated synclist for exclude policy.

ex:
Given a synclist distro: `/api/automation-hub/content/5910538-synclist/`

Collections served at: `/api/automation-hub/content/5910538-synclist/v3/collections/`
Excludes served at: `/api/automation-hub/content/5910538-synclist/v3/excludes/`

Accept: application/json
```json
{
    "collections": [
        {
            "name": "newswangerd.collection_demo"
        },
        {
            "name": "fernandomorais.kubernetes_fork"
        }
    ]
}
```

Accept: application/yaml
```yaml
collections:
- name: newswangerd.collection_demo
- name: fernandomorais.kubernetes_fork
```

Issue: AAH-606

### CODE REVIEW

**Comment by rochacbruno** [comment]
@gerrod3 @bmbouter this is the endpoint we are going to deploy to cloud once @awcrosby and @newswangerd review it.

File: galaxy_ng/app/api/v3/views/excludes.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        return synclist.collections.all()
+    except ObjectDoesNotExist:
+        return None
+
+
+def serialize_collection_queryset(queryset):
+    """Serialize a Queryset in to a JSONable format."""
+    return queryset is not None and [
+        {
+            "name": "{collection.namespace}/{collection.name}".format(collection=collection)
```
Comment (newswangerd) [inline galaxy_ng/app/api/v3/views/excludes.py:31]: The requirements.yml format uses `namespace.name`, not `namespace/name`.

**Review by newswangerd — CHANGES_REQUESTED** [review]

File: galaxy_ng/app/api/v3/views/excludes.py (hunk 1 above)
Comment (rochacbruno) [reply galaxy_ng/app/api/v3/views/excludes.py]: @newswangerd fixed

**Review by newswangerd — APPROVED** [review]

**Comment by newswangerd** [comment]
lgtm. Nice work!

**Pull request merged**